Docs: Add `DRACOExporter` page.

epreston opened the pull request:
Related issue: none.

**Description**

Creates a documentation page, DRACOExporter.html,  linking to the example and source code.  Adds a small comment to note the `exportColor` options parameter in the DRACOExporter.js source file.

epreston committed:
Create DRACOExporter.html

Create documentation for DRACOExporter

epreston committed:
Update DRACOExporter.js

add missing option "exportColor" to header comment.

Mugen87 commented:
If you add a new documentation page, please add it as an entry in `list.json`.`DRACOExporter` should be placed above this line:
https://github.com/mrdoob/three.js/blob/e2fe2af8f19fe70f3d2752ba4c2b40f05e3f7417/docs/list.json#L395

epreston commented:
I was looking for that, thank you.

epreston committed:
Update list.json

Add entry for English version of DRACOExporter in list.json since this is a new documentation page.

Mugen87 commented on docs/examples/en/exporters/DRACOExporter.html:
```diff
@@ -0,0 +1,74 @@
+
+		<h2>Methods</h2>
+
+		<h3>[method:Int8Array parse]( [param:Mesh|Points object], [param:Object options] )</h3>
```
```
[param:Mesh|Points object]
```
These multi-type definitions can't be parsed right now. Use the following as a workaround:
```
[param:Object3D object]
```

epreston replied:
Ok.  I'm looking for another example to see what it looks like formatted.  Does it look ok to you ?  I can work on the parser or more docs.  What's needed more ?

epreston replied:
I've made the update.

epreston committed:
Update DRACOExporter.html

Update formatting of multi-type definition.

Mugen87 commented on docs/examples/en/exporters/DRACOExporter.html:
```diff
@@ -0,0 +1,74 @@
+		<h3>[method:Int8Array parse]( [param:Mesh object] | [param:Points object], [param:Object options] )</h3>
+
+		<p>
+		[page:Mesh|Points object] — Mesh or Points to encode.<br />
```
This one also needs an update.

epreston replied:
Thank you.  I've written down to check this carefully.

epreston committed:
Update DRACOExporter.html

Update formatting of all multi-type definitions.

Mugen87 commented on docs/examples/en/exporters/DRACOExporter.html:
```diff
@@ -0,0 +1,74 @@
+			textures, animation, or node hierarchies – to use these features, embed Draco geometry
+			inside of a glTF file. A normal glTF file can be converted to a Draco-compressed glTF file
+			using [link:https://github.com/AnalyticalGraphicsInc/gltf-pipeline glTF-Pipeline].
+		</p>
```
This section makes sense for `DRACOLoader` but not necessarily for the exporter. Users don't need `glTF-Pipeline` if they export to drc via `DRACOExporter`.

epreston replied:
Need more advice on this one:

That makes sense, happy to ditch the paragraph. I'm worried about a misconception that made me leave it in.

This exporter won't help with updating ".glTF" files to contain draco encoded geometry buffers directly.  That's what everyone wants to do.  Like you said, this exporter only produces ".drc" files.  To draco encode parts of a glTF, they would still need the pipeline.  Draco compressing a blob of data within a glTF, setting the extention attributes, embedding the correct meta, that's far beyond what this thing can do.

Mugen87 replied:
You're right. After reading the paragraph again, it might be good to leave it in to avoid DRACO compression in context of glTF.
